packages/create-cloudflare/templates/queues/ts/src/index.ts

```diff
@@ -11,11 +11,6 @@
  * Learn more at https://developers.cloudflare.com/workers/
  */
 
-export interface Env {
```

Comment:
I don't think that this is correct 😕 

![Screenshot 2024-01-17 at 11 07 28](https://github.com/cloudflare/workers-sdk/assets/61631103/875694e9-da19-46d3-92d6-686d796be202)